BDArmory/Utils/VectorUtils.cs

```diff
@@ -342,6 +343,39 @@ public static bool CheckClearOfSphere(Ray ray, Vector3 sphereCenter, float spher
             }
         }
 
+        /// <summary>
+        /// Get angle between two pre-normalized vectors.
+        /// 
+        /// This implementation assumes that the input vectors are already normalized,
+        /// skipping such checks and normalization that Vector3.Angle does.
+        /// </summary>
+        /// <param name="from">First vector.</param>
+        /// <param name="to">Second vector.</param>
+        /// <returns>The angle between the two vectors.</returns>
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static float AnglePreNormalized(Vector3 from, Vector3 to)
+        {
+            float num2 = Mathf.Clamp(Vector3.Dot(from, to), -1f, 1f);
+            return Mathf.Acos(num2) * 57.29578f;
+        }
+
+        /// <summary>
+        /// Get angle between two pre-normalized vectors.
```

Comment:
This summary for `NormalizedDiff` is the same as for `AnglePreNormalized`.

Reply:
Fixed